src/swift/Dispatch.swift

```diff
@@ -67,7 +67,7 @@ public struct DispatchQoS : Equatable {
 
 	public static let unspecified = DispatchQoS(qosClass: .unspecified, relativePriority: 0)
 
-	public enum QoSClass {
+	public enum QoSClas: Sendable {
```

Comment:
I think you accidentally deleted last `s` here - QoSClass

Reply:
Heh, I did, thank you!